Zigbee: fix error when using VERSION file in Zigbee project.

morgendagen opened the pull request:
When using a `VERSION` file to keep track of the version of a Zigbee application, `zb_add_ota_header.py` exits with an error because the `APPLICATION_VERSION_STRING` contains a version tweak, e.g. 1.0.0+3.

The issue has been raised in https://devzone.nordicsemi.com/f/nordic-q-a/102654/mcuboot-build-numbers-in-ncs/440876.

NordicBuilder commented:
Thank you for your contribution!
It seems you are not a member of the nrfconnect GitHub organization. External contributions are handled as follows:
Large contributions, affecting multiple subsystems for example, may be rejected if they are complex, may introduce regressions due to lack of test coverage, or if they are not consistent with the architecture of nRF Connect SDK.
PRs will be run in our continuous integration (CI) test system.
If CI passes, PRs will be tagged for review and merged on successful completion of review. You may be asked to make some modifications to your contribution during review.
If CI fails, PRs may be rejected or may be tagged for review and rework.
PRs that become outdated due to other changes in the repository may be rejected or rework requested.
External contributions will be prioritized for review based on the relevance to current development efforts in nRF Connect SDK.  Bug fix PRs will be prioritized.
You may raise issues or ask for help from our Technical Support team by visiting https://devzone.nordicsemi.com/.

*Note: This comment is automatically posted and updated by the Contribs GitHub Action.*

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/nrfconnect/sdk-nrf?pullRequest=17424) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/nrfconnect/sdk-nrf?pullRequest=17424) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/nrfconnect/sdk-nrf?pullRequest=17424) it.</sub>

wiba-nordic reviewed (approved):
Documentation change looks good.

edmont reviewed (approved):
Thanks!

morgendagen commented:
I have squashed the two commits and fixed the commit message to comply with the guidelines. I am not sure if I should fix the error coming from pylint since it is unrelated to this PR.

PerMac commented:
@carlescufi could you please help resolving the compliance issue (unrelated to the PR)?
